core/score/agg_score.py

```diff
@@ -28,30 +28,25 @@
     ]
 
 AGG_FUNC = {
-    'coauthor': np.any,
-    'influenced_tot': np.nansum,
-    'influencing_tot': np.nansum,
-    'influenced_nsc': np.nansum,
-    'influencing_nsc': np.nansum,
+    'coauthor': 'any',
+    'influenced_tot': 'sum',
+    'influencing_tot': 'sum',
+    'influenced_nsc': 'sum',
+    'influencing_nsc': 'sum',
```

Comment:
Use Pandas builtin aggregation functions instead of passing callables. This happens to be faster 🤷‍♂️. Note that `nansum` is not an aggregation function Pandas recognizes. Instead, we use `.sum` after replacing all NaNs with zeros.